applications/ecallmgr/src/call_cmd/ecallmgr_call_command.erl

```diff
@@ -1514,3 +1515,47 @@ normalize_event_action_char(C) when is_integer(C), $A =< C, C =< $Z -> C + 32;
 normalize_event_action_char(C) when is_integer(C), 16#C0 =< C, C =< 16#D6 -> C + 32; % from string:to_lower
 normalize_event_action_char(C) when is_integer(C), 16#D8 =< C, C =< 16#DE -> C + 32; % so we only loop once
 normalize_event_action_char(C) -> C.
+
+maybe_set_init_whisper_mode(Node, UUID, JObj) ->
+    case kz_json:get_ne_binary_value(<<"Init-Whisper-Mode">>, JObj) of
+        'undefined' -> 'ok';
+        Mode -> set_init_whisper_mode(Node, UUID, Mode)
+    end.
+
+-spec set_init_whisper_mode(atom(), kz_term:ne_binary(), binary())  -> 'ok'.
+set_init_whisper_mode(Node, UUID, Mode) ->
+    case Mode of
+        <<"a">> ->
+            eavesdrop_bridge_both(Node, UUID),
+            eavesdrop_whisper_a(Node, UUID);
+        <<"b">> ->
+            eavesdrop_bridge_both(Node, UUID),
+            eavesdrop_whisper_b(Node, UUID);
+        <<"both">> ->
+            eavesdrop_bridge_both(Node, UUID),
+            eavesdrop_whisper_a(Node, UUID),
+            eavesdrop_whisper_b(Node, UUID);
+        Other ->
+            lager:debug("error bad value of Init-Whisper-Mode: ~p", [Other]),
+            'ok'
+    end.
+
+-spec eavesdrop_bridge_both(atom(), kz_term:ne_binary())  -> 'ok'.
+eavesdrop_bridge_both(Node, UUID) ->
+    _ = ecallmgr_util:send_cmd(Node, UUID, "unset", "eavesdrop_bridge_aleg"),
+    _ = ecallmgr_util:send_cmd(Node, UUID, "set", "eavesdrop_bridge_aleg=true"),
+    _ = ecallmgr_util:send_cmd(Node, UUID, "unset", "eavesdrop_bridge_bleg"),
+    _ = ecallmgr_util:send_cmd(Node, UUID, "set", "eavesdrop_bridge_bleg=true"),
```

Comment:
all these are expensive operations. it implies 4 calls to mod_kazoo, not a good thing